src/styles/blocks/reset.scss

```diff
@@ -0,0 +1,19 @@
+* {
+  margin: 0;
+
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  --nav-color: #060b35;
+  --hover-color: #34568b;
+  --link-color: #00acdc;
+  --shadow-box: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
```

Comment:
create separate file for variables, and use scss feature of creating variables(starting with $)